Resources/public/js/views/subitem/ez-subitemgridview.js

```diff
@@ -292,6 +293,21 @@ YUI.add('ez-subitemgridview', function (Y) {
         },
 
         /**
+         * Return an array containing the subitems that we want to append
+         * in the grid view.
+         *
+         * @method _getSubItemsToAppend
+         * @protected
+         * @param {EventFacade} e
+         */
+        _getSubItemsToAppend: function (e) {
```

Comment:
what you actually want is an array containing the newly added subitems, so this method would be better named `_getNewlyAddedSubitems` and should directly receive the `previousValue` and the `newValue`. You should also mark it as private, since it's really some internal plumbing.